vue/components/molecules/restrictions-alert/restrictions-alert.vue

```diff
@@ -10,11 +10,27 @@
                         {{ "ripe_commons.restrictions_alert.close_button" | locale }}
                     </a>
                 </div>
-                <div class="message-container">
-                    <span>
-                        {{ "ripe_commons.restrictions_alert.limited" | locale }}

```

Comment:
Let this stay inline, we usually let the static text be in the HTML. This is one of those scenarios: don't fix it if it ain't broken.